authz/sdk_end2end_test.go

```diff
@@ -21,13 +21,16 @@ package authz_test
 import (
 	"context"
 	"io"
+	"io/ioutil"
```

Comment:
FYI: the `ioutil` package is deprecated as of Go 1.17; you may wish to refactor to use other functions instead.

Reply:
I had to revert the changes to use `ioutil` instead of `os` as this resulted in test failures.
authz/sdk_server_interceptors.go:136:25: undefined: os.ReadFile
Some test targets do not use the latest version
`go version go1.15.15 linux/amd64` in Testing/vet-proto
`os` package at that version does not contain `ReadFile`, `WriteFile` API
https://pkg.go.dev/os@go1.15.15

I looked for alternative APIs (like os.Open followed by file.Read), but that involved reading the file chunk by chunk, which does not fit our use-case.